### PR TITLE
Fix twitter.com popup

### PR DESCRIPTION
**Pull request by piquark6046**
### How to reproduce
Just scroll the website down at any thread.

---

https://github.com/AdguardTeam/AdguardFilters/issues/109142

### CODE REVIEW

**Comment by piquark6046** [comment]
Example: https://twitter.com/github/status/1425505817827151872?cxt=HHwWgMC4jaP_tMgnAAAA

File: AnnoyancesFilter/sections/popups.txt
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:2191]: This rule is still required
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt
Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt:2191]: Big popup cannot reproduce.
@AdguardTeam/filters-maintainers Please take a look. Maybe can be blocked by scriptlet(`prevent-addEventListener` on scroll event, or something else)

Comment (piquark6046) [reply AnnoyancesFilter/sections/popups.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/112574#discussion_r824813341
Okay. I pushed.

File: AnnoyancesFilter/sections/popups.txt
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:122]: The first rule does not applied on my end.
What about this?
```
twitter.com#$?#div[data-testid="sheetDialog"]:upward(div[role="group"]) { display: none !important; }
twitter.com#$?#html { overflow: auto !important; padding-right: 0 !important; }
```
If you have account, please test with/without account in new private tabs.

**Review by Alex-302 — COMMENTED** [review]
Better move below to the rules with comments, and add a comment about purpose of these rules

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**
